learntools/ml-insights/nbs/tut2.ipynb

```diff
@@ -0,0 +1,446 @@
+       "        </tr>\n",
+       "    \n",
+       "        <tr style=\"background-color: hsl(0, 100.00%, 84.94%); border: none;\">\n",
+       "            <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
+       "                -0.0500\n",
+       "                \n",
+       "                    &plusmn; 0.0500\n",
+       "                \n",
+       "            </td>\n",
+       "            <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
+       "                Ball Possession %\n",
+       "            </td>\n",
+       "        </tr>\n",
+       "    \n",
+       "    \n",
+       "    </tbody>\n",
+       "</table>\n",
+       "    \n",
+       "\n",
+       "    \n",
+       "\n",
+       "\n",
+       "    \n",
+       "\n",
+       "    \n",
+       "\n",
+       "    \n",
+       "\n",
+       "    \n",
+       "\n",
+       "    \n",
+       "\n",
+       "    \n",
+       "\n",
+       "\n",
+       "\n"
+      ],
+      "text/plain": [
+       "<IPython.core.display.HTML object>"
+      ]
+     },
+     "execution_count": 54,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "import eli5\n",
+    "from eli5.sklearn import PermutationImportance\n",
+    "\n",
+    "perm = PermutationImportance(my_model, random_state=1).fit(val_X, val_y)\n",
+    "eli5.show_weights(perm, feature_names = val_X.columns.tolist())"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Interpreting Permutation Importances\n",
+    "\n",
+    "The values towards the top are the most important features, and those towards the bottom matter least.\n",
+    "\n",
+    "The first number in each row is a measure of how much the model performance decreased with a random shuffling. Like most things in data science, there is some randomness to the exact performance change measured from a shuffling a column.  For that reason, our permutation importance calculation is measured from multiple shuffles.  The number after the **±** is a measure of how much performance varied from one-reshuffling to the next.\n",
+    "\n",
+    "You'll occasionally see negative values for permutation importances. That is, in some cases the predictions on the shuffled (or noisy) data happened to be more accurate than the real data. The underlying cause for this is that the feature didn't matter (should have had an importance close to 0), but random chance caused the predictions on shuffled data to be more accurate. This is especially likely in smaller datasets, such as the one used in this example, because there is more room for luck/chance.\n",
+    "\n",
+    "In this case, we this most important feature was **Goals scored**. A team is more likely to have the \"Man of the Game\" if they score a lot of goals. That seems sensible, even to a non-fan. Soccer fans may enjoy pondering explanations for all of the features.\n",
```

Comment:
> A team is more likely to have the "Man of the Game" if they score a lot of goals.

That may be true, but I don't think we can infer that from the permutation importance results, can we?